src/ServiceControl.Monitoring/Settings.cs

```diff
@@ -8,16 +8,34 @@ namespace ServiceControl.Monitoring
     using System.Threading.Tasks;
     using NLog;
     using Transports;
+    using ServiceControl.Monitoring.Infrastructure.Settings;
 
     //TODO: align names with SC and SC.Audit
     public class Settings
     {
+        public Settings()
+        {
+            TryLoadLicenseFromConfig();
+
+            TransportType = SettingsReader<string>.Read("TransportType");
```

Comment:
the removed code used `TransportType = reader.Read<string>("Monitoring/TransportType"),` with the monitoring prefix (some other options seem to do that too), is that an intentional change?

Reply:
Yeah these were changes that came in from 

https://github.com/Particular/ServiceControl/commit/cac7201349f4af9ffffa8f81aa2d19403fb2c007#diff-b616500841627c3849f0e83080ac2d844bbfdf95abd9f255bb7f6129c674fb12

Force pushed